packages/react-examples/src/react-badge/Badge/BadgeChildren.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { Badge, BadgeProps } from '@fluentui/react-badge';
+
+const BadgeChildrenExamples = (props: BadgeProps) => (
```

Comment:
```suggestion
const BadgeContentExamples = (props: BadgeProps) => (
```

nit: I would rather name these stories as "content"

Reply:
Done